### ISSUE TITLE
Reproducibility

### ISSUE THREAD

**Issue by minkyujeon** [OliverRensu/xAR#15]
Hello, I trained the xAR-base model for 400 epochs and evaluated it using the checkpoint.
According to the paper, the reported FID is 2.48, but I obtained an FID of 3.1128.
Could you please let me know if I missed something?
Below are the training and evaluation commands:

Train:
> srun --ntasks=$SLURM_NNODES --ntasks-per-node=1 --cpus-per-task=$SLURM_CPUS_PER_TASK bash -lc "
  echo Running on host: \$(hostname)
  echo MASTER_ADDR=$MASTER_ADDR MASTER_PORT=$MASTER_PORT NODE_RANK=\$SLURM_NODEID
  torchrun \
    --nnodes=$SLURM_NNODES \
    --nproc_per_node=4 \
    --node_rank=\$SLURM_NODEID \
    --master_addr=$MASTER_ADDR \
    --master_port=$MASTER_PORT \
    main_mar.py \
      --img_size 256 \
      --vae_path /scratch/gpfs/ZHONGE/mj7341/github_repos/mar/pretrained_models/vae/kl16.ckpt \
      --vae_embed_dim 16 --vae_stride 16 --patch_size 1 \
      --model xar_base \
      --epochs 400 --warmup_epochs 100 --batch_size 64 --blr 5e-5 \
      --output_dir $OUTPUT_DIR \
      --data_path $IMAGENET_PATH \
"

Eval:

> torchrun --nproc_per_node=4 --master_port=6781 \
    eval.py \
    --model xar_base \
    --eval_bsz 256 --num_images 50000 \
    --num_iter 256 --num_steps 40  --cfg 2.7 \
    --output_dir $OUTPUT_DIR \
    --resume $RESUME --vae_path $VAE \
    --data_path $IMAGENET_PATH --evaluate

**Comment by OliverRensu** [comment]
Please check the learning-rate decay in util/lr_sched.py. I configured the scheduler for 800 epochs. If you’re only training for 400 epochs, change the 800/600 milestones to 400/300.
